app/config/sources.inc.php

```diff
@@ -414,6 +414,7 @@
     'whatsnew/whatsnew_android_44.lang',
     'whatsnew/whatsnew_android_45.lang',
     'whatsnew/whatsnew_android_46_beta.lang',
+    'whatsnew/whatsnew_ios_2_1.lang',
```

Comment:
Ah, I thought it was for the version we're localizing now. Also completely missed that the minor update for 2.0 was called 2.1